eox_nelp/pearson_vue/utils.py

```diff
@@ -40,5 +40,7 @@ def generate_client_authorization_id(user_id: int, course_id: str) -> str:
         str: string that represents the client_auth_id
     """
     course_enrollment = CourseEnrollment.objects.get(user_id=user_id, course_id=course_id)
+    anonymous_user_id = anonymous_id_for_user(course_enrollment.user, course_id)
+    anonymous_user_id_instance = AnonymousUserId.objects.get(anonymous_user_id=anonymous_user_id)
```

Comment:
Why not simply use the AnonymousUserId alone for identification? I believe it's unique per course and user. However, if some AnonymousUserId values are missing, then perhaps a combination of course_enrollment_id and anonymous_user_id could be used for tracking. That is is the reason?

Reply:
Same CourseEnrollment reason, if someone deletes the record we won't be able to track the user